packages/framework/tests/Unit/Model/Customer/TestCustomerProvider.php

```diff
@@ -47,6 +47,7 @@ public static function getTestCustomerUserData(bool $isCompany = true): Customer
         $customerUserData->firstName = 'Firstname';
         $customerUserData->lastName = 'Lastname';
         $customerUserData->email = 'no-reply@shopsys.com';
+        $customerUserData->telephone = 'telephone';
```

Comment:
Hmm, this indeed fixes the test but it reveals that we are missing a test for the social network login scenario:

1. a customer user does not have saved any nullable data (basically he has email only, the rest is null)
2. he creates an order where he fills in all the data
3. assert customer user data are amended with the order data

Could you please add such a test?

Btw, while testing this, I noticed the data update still does not work properly for the billing address data (I registered via google, created an order but in my profile, there are still no billing data). I did not try it with the delivery address data.

I believe that at least one of the problems is in `RegistrationDataFactory::createFromSocialNetworkProfile()` where empty strings are still used.